### PR TITLE
Create model `Poll`, add foreign keys and rename related fields into `entity`

### PR DESCRIPTION
**Pull request by amatissart**
First step, described in #526 

Note that contrary to what was described initially, the foreign keys do not need to be created with `null=True`. Their default value is specified with a method to find the default poll.

The `name` of this default poll is to be confirmed: `videos` for now.

### CODE REVIEW

File: backend/tournesol/admin.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ class EntityCriteriaScoreAdmin(admin.ModelAdmin):
 class ContributorRatingAdmin(admin.ModelAdmin):
```
Comment (GresilleSiffle) [inline backend/tournesol/admin.py:49]: future considerations

we could add a filter on entity type and and filter on poll in this admin

File: backend/tournesol/models/ratings.py [hunk 2]
```diff
@@ -25,12 +26,18 @@ class ContributorRating(models.Model, WithFeatures):
...
     is_public = models.BooleanField(
         default=False, null=False, help_text="Should the rating be public?"
     )
 
     class Meta:
-        unique_together = ["user", "video"]
+        unique_together = ["user", "entity", "poll"]
 
     def __str__(self):
         return "%s on %s" % (self.user, self.video)
```
Comment (GresilleSiffle) [inline backend/tournesol/models/ratings.py:43]:
```suggestion
        return "%s on %s" % (self.user, self.entity)
```

File: backend/tournesol/models/entity_score.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        validators=[MinValueValidator(0.0), MaxValueValidator(1.0)],
+        help_text="Top quantile for all rated videos for aggregated scores"
+        "for the given criteria. 0.0=best, 1.0=worst",
+    )
+
+    class Meta:
+        unique_together = ["video", "poll", "criteria"]
+
+    def __str__(self):
+        return f"{self.entity}/{self.criteria}/{self.score}"
```
Comment (GresilleSiffle) [inline backend/tournesol/models/entity_score.py:57]: We refer to `self.entity` here but the model has still a `video` attribute. Maybe a migration is missing?

File: backend/tournesol/models/ratings.py (hunk 2 above)
Comment (amatissart) [reply backend/tournesol/models/ratings.py]: Should be fixed in ffdeff7

File: backend/tournesol/models/entity_score.py (hunk 3 above)
Comment (amatissart) [reply backend/tournesol/models/entity_score.py:57]: Should be fixed in ffdeff7b

**Review by GresilleSiffle — APPROVED** [review]

**Pull request merged**